transport/tchannel/channel_outbound.go

```diff
@@ -103,10 +110,15 @@ func (o *ChannelOutbound) IsRunning() bool {
 
 // Call sends an RPC over this TChannel outbound.
 func (o *ChannelOutbound) Call(ctx context.Context, req *transport.Request) (*transport.Response, error) {
+	span := opentracing.SpanFromContext(ctx)
```

Comment:
This is not the right tchannel outbound span.
The tchannel outbound span is created when`xxx.BeginCall(..)` is invoked, and the [span is created in tchannel-go](https://github.com/uber/tchannel-go/blob/dev/outbound.go#L120).

It seems that tchannel doesn't expose the outbound span currently. We probably need to expose it from tchannel-go first.

Reply:
I agree. I will make this PR solely deal with inbound error tagging. For outbound, I will implement after exposing it from tchannel-go.